src/AnalyzeView/MAVLinkInspectorController.cc

```diff
@@ -222,10 +223,7 @@ void
 QGCMAVLinkMessage::update(mavlink_message_t* message)
 {
     _count++;
-    //-- If we are not consuming this message, no need to parse it
-    if(!_selected && !_fieldSelected) {
-        return;
-    }
```

Comment:
@dogmaphobic I assume you put this in for performance. But it makes the inspector either not shows the user what they want to see, or in other cases it shows wrong data. Was there another reason other than perf for this?

Reply:
Yes, it's hundreds of thousands signal/slot calls each.

Reply:
(per second)

Reply:
Fixed https://github.com/mavlink/qgroundcontrol/pull/8991